cairo_vm_hints/src/hint_processor/print.rs

```diff
@@ -0,0 +1,118 @@
+use super::CustomHintProcessor;
+use cairo_vm::hint_processor::builtin_hint_processor::hint_utils::{
+    get_integer_from_var_name, get_relocatable_from_var_name,
+};
+use cairo_vm::{
+    hint_processor::builtin_hint_processor::builtin_hint_processor_definition::HintProcessorData,
+    types::exec_scope::ExecutionScopes,
+    vm::{errors::hint_errors::HintError, vm_core::VirtualMachine},
+    Felt252,
+};
+use std::collections::HashMap;
+
+pub const HINT_PRINT_HASH: &str = "from tools.py.hints import print_u256\nprint(\"\\n\")\nprint_u256(ids.block_header_hash_little,f\"block_header_keccak_hash_{ids.index}\")\nprint_u256(ids.expected_block_hash,f\"expected_keccak_hash_{ids.index}\")";
+pub const HINT_PRINT_FINAL: &str = "print(\"new root poseidon\", ids.new_mmr_root_poseidon)\nprint(\"new root keccak\", ids.new_mmr_root_keccak.low, ids.new_mmr_root_keccak.high)\nprint(\"new size\", ids.mmr_array_len + ids.mmr_offset)";
```

Comment:
I would use multiline string here and in all const string for better readability. (Like we did in eth_essentials)